lenskit-eval/src/main/java/org/lenskit/eval/temporal/TemporalEvaluator.java

```diff
@@ -217,8 +250,42 @@ public void execute() throws IOException, RecommenderBuildException {
                 if (n > 0) {
                     rmse = sqrt(sse / n);
                 }
-                //writes the Prediction Score and TARMSE on file
-                tableWriter.writeRow(r.getUserId(), r.getItemId(), r.getValue(), r.getTimestamp(), predict, rmse);
+
+                //calculate recommendation rank
+                int rank = -1;
+                LongSet itemsInDao = new LongOpenHashSet(limitedDao.getItemIds().size());
```

Comment:
Can just say `new LongOpenHashSet(limitedDao.getItemIds())` and it will create a new copy of the item ID set.